packages/commonwealth/client/scripts/views/SublayoutHeader.tsx

```diff
@@ -85,7 +84,7 @@ export const SublayoutHeader = ({ onMobile }: SublayoutHeaderProps) => {
           {isWindowSmallInclusive(window.innerWidth) && (
             <CWDivider isVertical />
           )}
-          {(featureFlags.sidebarToggle || onMobile) && app.activeChainId() && (
```

Comment:
shouldn't we remove `featureFlags.sidebarToggle` altogether? from webpack and everywhere